From 6738700c5ddc051624d85bc75e0f34e27e18eaa2 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Mon, 15 Jan 2024 10:26:05 +0100
Subject: [PATCH 0879/1141] arm64: dts: st: remove backlight on stm32mp257f-ev1
 board

The dsi panel has been removed, the dsi backlight has to be removed too.

Change-Id: Ic291f9d12cfced0c74fe9757ccfdb917f9ec0eb0
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/351203
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 8 --------
 1 file changed, 8 deletions(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -95,14 +95,6 @@
 		reg = <0x0 0x80000000 0x1 0x0>;
 	};
 
-	panel_dsi_backlight: panel-dsi-backlight {
-		compatible = "gpio-backlight";
-		gpios = <&gpioi 5 GPIO_ACTIVE_LOW>;
-		default-on;
-		default-brightness-level = <0>;
-		status = "disabled";
-	};
-
 	panel_lvds: panel-lvds {
 		compatible = "edt,etml0700z9ndha", "panel-lvds";
 		enable-gpios = <&gpiog 15 GPIO_ACTIVE_HIGH>;
